daemon/src/maker_cfd.rs

```diff
@@ -338,18 +344,23 @@ impl Actor {
         self.cfd_feed_actor_inbox
             .send(load_all_cfds(&mut conn).await?)?;
 
-        // let latest_announcement = self
-        //     .latest_announcement
-        //     .to_owned()
-        //     .context("Unaware of oracle's latest announcement.")?;
-
-        // self.oracle_actor
-        //     .do_send_async(oracle::MonitorEvent {
-        //         event_id: latest_announcement.id,
-        //     })
-        //     .await?;
+        let offer_announcements = self
+            .latest_announcements
+            .clone()
+            .context("No oracle announcements available")?;
+        let offer_announcement = offer_announcements
+            .0
+            .iter()
+            .find(|announcement| announcement.id == cfd.order.oracle_event_id)
+            .context("Order's announcement not found in list of current oracle announcements")?;
```

Comment:
❓ I can't see the types here, so maybe I'm missing something, but can't you just use the `cfd.order.oracle_event_id` directly when sending the `oracle::MonitorEvent` message to the oracle actor? The maker shouldn't need to verify anything about an offer _they_ created.

Reply:
Okay, ignore this. You need to get the full announcement to get the `nonce_pks`.

Reply:
We need the actual `Announcement` to retrieve the `nonce_pks` after we send the message to the monitor actor. 
In the `Order` we only have the `OracleEventId`. I think it is safer to only trigger the monitoring once we know we also have the announcement (containing the `nonce_pks`), that's why I first retrieve the announcement and only if that succeeded go on with triggering monitoring.